frontend/apps/picasso/stores/defi/polkadot/balances/PolkadotBalancesUpdater.tsx

```diff
@@ -156,17 +156,17 @@ const PolkadotBalancesUpdater = ({
                   picassoBalanceSubscriber(chain, asset, chainId);
                   break;
                 case "karura":
-                  fetchKaruraBalanceByAssetId(
-                    chain.parachainApi!,
-                    chain.accounts[selectedAccount].address,
-                    String(asset.meta.symbol)
-                  ).then((balance) => {
-                    updateAssetBalance({
-                      substrateNetworkId: chainId as SubstrateNetworkId,
-                      assetId: asset.meta.assetId,
-                      balance
-                    });
-                  });
+                  // fetchKaruraBalanceByAssetId(
+                  //   chain.parachainApi!,
+                  //   chain.accounts[selectedAccount].address,
+                  //   String(asset.meta.symbol)
+                  // ).then((balance) => {
+                  //   updateAssetBalance({
+                  //     substrateNetworkId: chainId as SubstrateNetworkId,
+                  //     assetId: asset.meta.assetId,
+                  //     balance
+                  //   });
+                  // });
```

Comment:
NIT:
Remove?

Reply:
Yes, it is fixed in my PR, leave it as it is.

Reply:
Let's bring this code back, the issue is not here but elsewhere in the initialization.